cognite_toolkit/cdf_tk/load.py

```diff
@@ -684,19 +729,21 @@ def load_resource(self, filepath: Path, dry_run: bool) -> TimeSeries | TimeSerie
 
 
 @final
-class TransformationLoader(Loader[str, Transformation, TransformationList]):
+class TransformationLoader(Loader[str, Transformation, Transformation, TransformationList, TransformationList]):
     api_name = "transformations"
     folder_name = "transformations"
     filename_pattern = (
         r"^(?:(?!\.schedule).)*$"  # Matches all yaml files except file names who's stem contain *.schedule.
     )
     resource_cls = Transformation
+    resource_write_cls = Transformation
     list_cls = TransformationList
+    list_write_cls = TransformationList
     dependencies = frozenset({DataSetsLoader, RawLoader})
 
     @classmethod
     def get_required_capability(cls, ToolGlobals: CDFToolConfig) -> Capability:
-        scope = (
+        scope: capabilities.AllScope | capabilities.DataSetScope = (
```

Comment:
Didn't we introduce space and extraction pipeline scope?

Reply:
apparently not.